packages/cli/lib/LocalDevManager.js

```diff
@@ -265,23 +358,32 @@ class LocalDevManager {
     this.standbyChanges.push(changeInfo);
   }
 
-  async sendChanges({ event, filePath, remotePath }) {
+  async sendChanges(changeInfo) {
```

Comment:
Nit: You could destructure changeInfo and make this slightly easier to read.

Reply:
Good idea, done in [bbd694d](https://github.com/HubSpot/hubspot-cli/pull/835/commits/bbd694dec58a8f0cecad270b5f5135a0753efcb8) 👍